common/djangoapps/student/helpers.py

```diff
@@ -609,7 +612,11 @@ def do_create_account(form, custom_form=None):
         email=form.cleaned_data["email"],
         is_active=False
     )
-    user.set_password(form.cleaned_data["password"])
+    if flag_is_active(get_current_request(), 'password_unicode_normalize'):
+        normalized_password = unicodedata.normalize('NFKC', form.cleaned_data["password"])
+        user.set_password(normalized_password)
+    else:
+        user.set_password(form.cleaned_data["password"])
     registration = Registration()
```

Comment:
I hope it's not obnoxious to do a drive-by review... I would find the logic easier to follow like this:
```
    password = form.cleaned_data["password"]
    if flag_is_active(get_current_request(), 'password_unicode_normalize'):
        password = unicodedata.normalize('NFKC', password)
    user.set_password(password)
```

Reply:
Hi Ned - not obnoxious, but I haven't really put this out for review yet and am still working on it! I appreciate the feedback anyway and will change this.